src/main/java/com/googleinterns/zoomtube/utils/CommentUtil.java

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.users.User;
+import com.googleinterns.zoomtube.data.Comment;
+import java.util.Date;
+import java.util.Optional;
+
+/** Provides methods to create Comment Entities and Comments. */
+public final class CommentUtil {
+  public static final String KIND = "Comment";
+  public static final String LECTURE = "lecture";
+  public static final String PARENT = "parent";
+  public static final String TIMESTAMP = "timestamp";
+  public static final String AUTHOR = "author";
+  public static final String CONTENT = "content";
+  public static final String CREATED = "created";
+
+  private CommentUtil(){};
```

Comment:
```suggestion
  private CommentUtil() {};
```

Reply:
nit: move this constructor at the bottom because it isn't important.

Reply:
The linter seems to remove the space automatically...

Reply:
Moved constructor to bottom